doc/changelogs/CHANGELOG_V6.md

```diff
@@ -2,6 +2,7 @@
 
 <!--lint disable prohibited-strings-->
 <!--lint disable maximum-line-length-->
+<!-- markdownlint-disable MD038 -->
```

Comment:
Are the markdownlint comments needed? I thought in this repo there's no markdownlint used.

Reply:
https://github.com/nodejs/node/blob/c246f813f8b1bbc8f0e13ab77452da4dff437635/Makefile#L1211

Reply:
I think remark-lint might be a better choice, but we should probably try it in nodejs.org repo after fixing most of the issues and see where we are at.

Reply:
Since markdownlint isn't added yet, I'd say just remove the comment when you are done with the patch.

Reply:
Just expressing my opinion, that's all :)

Unless they are used, i.e. markdownlint is added, they offer no value IMO. But I'll leave you guys decide.